src/Core/Types/Types/Descriptors/OutputFieldDescriptorBase.cs

```diff
@@ -55,6 +55,20 @@ protected void Type<TOutputType>()
                 TypeContext.Output);
         }
 
+        protected void Type(Type type)
+        {
```

Comment:
The above method should now lean on this new non-generic method.

Reply:
added